editor/components/block-settings-menu/block-unwrap.js

```diff
@@ -0,0 +1,69 @@
+/**
```

Comment:
Can we leave unwrapping to a separate PR. Not saying we shouldn't do it, but it seems better (at least for me) to discuss separately?

Reply:
Sure, but how do I take care of the block transforms in this PR? Or should I only add the unwrapping if the block has specified a transform (vs always offering unwrap if the block has innerBlocks)?

Reply:
This is a new feature in this PR right? Is it something needed to replace an existing feature? I suspect it's here to replace some transforms that are not possible in the nested approach? Maybe we don't care that much about these transforms at the moment?

Reply:
Yes, it's to take care of quote => paragraph/heading. See https://github.com/WordPress/gutenberg/pull/6054/files#diff-cda27cec5557c5b032f2b19c44309a1dL106. I added the unwrapping to show it's easier to take care of these kind of transformations. But I agree this should maybe be discussed more. Are you fine with dropping these transformations then at the moment?

Reply:
Okay I removed this for now.